kustomize/traffic/namespaceScoped/kustomization.yaml

```diff
@@ -1,5 +1,14 @@
-bases:
-- ../base
-
 resources:
-- roles.yaml
\ No newline at end of file
+- configmap.yaml
+- headlessservice.yaml
+- role.yaml
+- rolebinding.yaml
+- serviceaccount.yaml
+- statefulset.yaml
+
+namespace: default
```

Comment:
I think we should remove this line so that it will install in the namespace set in the current kubernetes context.